README.md

```diff
@@ -49,7 +49,7 @@ Consequently, the base URL would change to
 
 4. Once the server is up and running, you can start using the app. For ease of testing, follow these steps: 
 
-(1) post call: `python3 app.py post` (this call will return a drs object id, e.g. 12345)
+(1) post call: `action post` (this call will return a drs object id, e.g. 12345)
```

Comment:
Can you run this command as 
`action post` instead of `python3 app.py post`?

Reply:
Yes. That is why I updated the readme.